File: prempalsingh/fizzbuzz.f90
      program fizzbuzz
            integer :: number
            do number = 1, 100
                  if (mod(number,15) == 0) then
                        print *, 'FizzBuzz'
                  else if (mod(number,3) == 0) then
                        print *, 'Fizz'
                  else if (mod(number,5) == 0) then
                        print *, 'Buzz'
                  else
                        print *, number
                  end if
            end do
      end program fizzbuzz

File: daxlab/fizzbuzz.pas
program fizzbuzz;
var
  foo: integer;
begin
  for foo := 1 to 100 do
    if foo mod 15 = 0 then
      writeln('FizzBuzz')
    else if foo mod 3 = 0 then
      writeln('Fizz')
    else if foo mod 5 = 0 then
      writeln('Buzz')
    else
      writeln(foo)
end.
